src/main/scala/cromwell/binding/command/ParameterCommandPart.scala

```diff
@@ -90,6 +90,7 @@ case class ParameterCommandPart(wdlType: WdlType, name: String,
      * turned into strings, but a more sophisticated solution will be
      * needed for compound types */
     paramValueEvaluated match {
+      case str:WdlString if str.value.isEmpty => ""
```

Comment:
No, this doesn't have anything to do with null values.  I can try your suggestion, that looks cleaner.  This is basically to short circuit so we don't prepend the prefix if an empty value was specified for a parameter.